templates/UmbracoProject/.template.config/template.json

```diff
@@ -0,0 +1,329 @@
+          {
+            "regex": "\\\t",
+            "replacement": "\\\t"
+          }
+        ]
+      },
+      "replaces": "CONNECTION_STRING_FROM_TEMPLATE"
+    },
+    "HasConnectionString": {
+      "type": "computed",
+      "value": "(ConnectionString != \"\")"
+    },
+    "UsingUnattenedInstall": {
+      "type": "computed",
+      "value": "(UnattendedUserName != \"\" && UnattendedUserEmail != \"\" && UnattendedUserPassword != \"\" && ConnectionString != \"\")"
+    },
+    "NoNodesViewPath": {
+      "displayName": "No nodes view path",
+      "description": "Path to a custom view presented with the Umbraco installation contains no published content",
+      "type": "parameter",
+      "datatype": "string",
+      "defaultValue": ""
+    },
+    "NoNodesViewPathReplacer": {
+      "type": "generated",
+      "generator": "regex",
+      "dataType": "string",
+      "parameters": {
+        "source": "NoNodesViewPath",
+        "steps": [
+          {
+            "regex": "\\\\",
+            "replacement": "\\\\"
+          },
+          {
+            "regex": "\\\"",
+            "replacement": "\\\""
+          },
+          {
+            "regex": "\\\n",
+            "replacement": "\\\n"
+          },
+          {
+            "regex": "\\\t",
+            "replacement": "\\\t"
+          }
+        ]
+      },
+      "replaces": "NO_NODES_VIEW_PATH_FROM_TEMPLATE"
+    },
+    "HasNoNodesViewPath": {
+      "type": "computed",
+      "value": "(NoNodesViewPath != \"\")"
+    },
+    "PackageProjectName": {
+      "displayName": "Umbraco package project name",
+      "description": "The name of the package project this should be a test site for.",
+      "type": "parameter",
+      "datatype": "string",
+      "defaultValue": "",
+      "replaces": "PACKAGE_PROJECT_NAME_FROM_TEMPLATE"
+    },
+    "NamespaceReplacer": {
+      "type": "generated",
+      "generator": "regex",
+      "dataType": "string",
+      "parameters": {
+        "source": "name",
+        "steps": [
+          {
+            "regex": "\\s",
+            "replacement": "_"
+          },
+          {
+            "regex": "-",
+            "replacement": "_"
+          },
+          {
+            "regex": "^[^a-zA-Z_]+",
+            "replacement": "_"
+          }
+        ]
+      },
+      "replaces": "Umbraco.Cms.Web.UI"
+    },
+    "HttpPort": {
+      "type": "generated",
+      "generator": "port",
+      "parameters": {
+        "fallback": 5000
+      },
+      "replaces": "HTTP_PORT_FROM_TEMPLATE"
+    },
+    "HttpsPort": {
+      "type": "generated",
+      "generator": "port",
+      "parameters": {
+        "low": 44300,
+        "high": 44399,
+        "fallback": 5001
+      },
+      "replaces": "HTTPS_PORT_FROM_TEMPLATE"
+    },
+    "TelemetryId": {
+      "type": "generated",
+      "generator": "guid",
+      "parameters": {
+        "defaultFormat": "d"
+      },
+      "replaces": "TELEMETRYID_FROM_TEMPLATE"
+    }
```

Comment:
This now generates a unique telemetry ID (lowercase GUID) for each new project, as that was previously only done when using the attended install (and required unnecessary config manipulation).